src/manager.h

```diff
@@ -230,7 +230,7 @@ class Manager
 
   double                        target_loop_rate_hz_                = 0.0;
   bool                          zero_latency_required_              = true;
-  bool                          faulted_                            = false;
+  bool                          faulted_                            = true;
```

Comment:
Why start the manager faulted?

Reply:
The manager calls Process() and then the current wkc is checked against the expected wkc. If these differ and IsFaulted() returns false, we fault all devices. This makes sense during nominal operation but when ConfigFromYaml() is called Process() is called twice and I found the system would immediately error out due to wkc issues with my implementation.
Starting the system in faulted state causes the wkc check to be ignored and allows the two Process() calls to occur before ExecuteAllDeviceResets() is called (at the end of the ConfigFromYaml() method).
I found this to be the simplest change to get around this issue.